src/mission.rs

```diff
@@ -106,11 +107,15 @@ impl Mission {
                     .parent()
                     .expect("parent of a target folder is a root folder");
                 if add_all_src {
-                    let src_dir = item_path.join("src");
-                    if src_dir.exists() {
-                        directories_to_watch.push(src_dir);
-                    } else {
-                        warn!("missing src dir: {:?}", src_dir);
+                    let src_watch_iter = iter::once("str");
```

Comment:
"str" ?

(I'm fixing it, don't make a new commit)

Reply:
Oh wow, sorry about that! It seems str was on my mind. Thanks for fixing it!